src/Auth/DigestAuthenticate.php

```diff
@@ -276,7 +277,8 @@ protected function validNonce($nonce)
         if ($expires < microtime(true)) {
             return false;
         }
+        $check = hash_hmac('sha1', $expires . ':' . $this->getConfig('secret'), $this->getConfig('secret'));
```

Comment:
Is this just me, or shouldn't this be sha256? seems to be causing some trouble for me.